testData/solutions/win32.json5

```diff
@@ -6518,5 +6518,57 @@
                 "type": "gpii.deviceReporter.alwaysInstalled"
             }
         ]
+    },
+    "net.gpii.async": {
+        // This solution exists only to support GPII automated tests. It references the "async settings handler" which will
+        // trigger a wait of the given milliseconds during either the get and set settingsHandler action.
+        "name": "GPII Test solution for triggering an asynchronous settings handler on the get and set",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configuration": {
+                "type": "gpii.settingsHandlers.async",
+                "supportedSettings": {
+                    "timeInMs": {
+                        "schema": {
+                            "title": "The time to wait on calling the get, set or launch",
+                            "description": "The strategy to be used to simulate an asynchronous settings handler",
+                            "type": "number"
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "timeInMs": "http://registry\\.gpii\\.net/common/timeout"
```

Comment:
We don't need to go as far as implementing a term in the real registry for a parameter only of interest in test cases - this will excite comment! Let's just tweak the existing repository of mocks.